npm start fails

npm install worked:

```
⋊> ~/g/3/walletconnect.js on master ◦ npm install                       13:12:12

> deasync@0.1.12 install /home/ligi/git/3rd/walletconnect.js/node_modules/deasync
> node ./build.js

`linux-x64-node-6` exists; testing
Binary is fine; exiting

> parcel-bundler@1.7.0 postinstall /home/ligi/git/3rd/walletconnect.js/node_modules/parcel-bundler
> node -e "console.log('\u001b[35m\u001b[1mLove Parcel? You can now donate to our open collective:\u001b[22m\u001b[39m\n > \u001b[34mhttps://opencollective.com/parcel/donate\u001b[0m')"

Love Parcel? You can now donate to our open collective:
 > https://opencollective.com/parcel/donate
walletconnect.js@1.0.0 /home/ligi/git/3rd/walletconnect.js
└─┬ parcel-bundler@1.7.0 
  ├─┬ babel-code-frame@6.26.0 
  │ ├─┬ chalk@1.1.3 
  │ │ ├── ansi-styles@2.2.1 
  │ │ ├── has-ansi@2.0.0 
  │ │ ├─┬ strip-ansi@3.0.1 
  │ │ │ └── ansi-regex@2.1.1 
  │ │ └── supports-color@2.0.0 
  │ ├── esutils@2.0.2 
  │ └── js-tokens@3.0.2 
  ├─┬ babel-core@6.26.0 
  │ ├── babel-helpers@6.24.1 
  │ ├── babel-messages@6.23.0 
  │ ├─┬ babel-register@6.26.0 
  │ │ ├── core-js@2.5.4 
  │ │ ├─┬ home-or-tmp@2.0.0 
  │ │ │ ├── os-homedir@1.0.2 
  │ │ │ └── os-tmpdir@1.0.2 
  │ │ └─┬ source-map-support@0.4.18 
  │ │   └── source-map@0.5.7 
  │ ├─┬ babel-runtime@6.26.0 
  │ │ └── regenerator-runtime@0.11.1 
  │ ├── convert-source-map@1.5.1 
  │ ├─┬ debug@2.6.9 
  │ │ └── ms@2.0.0 
  │ ├── lodash@4.17.5 
  │ ├─┬ minimatch@3.0.4 
  │ │ └─┬ brace-expansion@1.1.11 
  │ │   ├── balanced-match@1.0.0 
  │ │   └── concat-map@0.0.1 
  │ ├── path-is-absolute@1.0.1 
  │ ├── private@0.1.8 
  │ ├── slash@1.0.0 
  │ └── source-map@0.5.7 
  ├─┬ babel-generator@6.26.1 
  │ ├─┬ detect-indent@4.0.0 
  │ │ └─┬ repeating@2.0.1 
  │ │   └─┬ is-finite@1.0.2 
  │ │     └── number-is-nan@1.0.1 
  │ ├── jsesc@1.3.0 
  │ ├── source-map@0.5.7 
  │ └── trim-right@1.0.1 
  ├─┬ babel-plugin-transform-es2015-modules-commonjs@6.26.0 
  │ └── babel-plugin-transform-strict-mode@6.24.1 
  ├─┬ babel-plugin-transform-react-jsx@6.24.1 
  │ ├── babel-helper-builder-react-jsx@6.26.0 
  │ └── babel-plugin-syntax-jsx@6.18.0 
  ├─┬ babel-preset-env@1.6.1 
  │ ├── babel-plugin-check-es2015-constants@6.22.0 
  │ ├── babel-plugin-syntax-trailing-function-commas@6.22.0 
  │ ├─┬ babel-plugin-transform-async-to-generator@6.24.1 
  │ │ ├── babel-helper-remap-async-to-generator@6.24.1 
  │ │ └── babel-plugin-syntax-async-functions@6.13.0 
  │ ├── babel-plugin-transform-es2015-arrow-functions@6.22.0 
  │ ├── babel-plugin-transform-es2015-block-scoped-functions@6.22.0 
  │ ├── babel-plugin-transform-es2015-block-scoping@6.26.0 
  │ ├─┬ babel-plugin-transform-es2015-classes@6.24.1 
  │ │ ├── babel-helper-define-map@6.26.0 
  │ │ ├── babel-helper-function-name@6.24.1 
  │ │ ├── babel-helper-optimise-call-expression@6.24.1 
  │ │ └── babel-helper-replace-supers@6.24.1 
  │ ├── babel-plugin-transform-es2015-computed-properties@6.24.1 
  │ ├── babel-plugin-transform-es2015-destructuring@6.23.0 
  │ ├── babel-plugin-transform-es2015-duplicate-keys@6.24.1 
  │ ├── babel-plugin-transform-es2015-for-of@6.23.0 
  │ ├── babel-plugin-transform-es2015-function-name@6.24.1 
  │ ├── babel-plugin-transform-es2015-literals@6.22.0 
  │ ├── babel-plugin-transform-es2015-modules-amd@6.24.1 
  │ ├─┬ babel-plugin-transform-es2015-modules-systemjs@6.24.1 
  │ │ └── babel-helper-hoist-variables@6.24.1 
  │ ├── babel-plugin-transform-es2015-modules-umd@6.24.1 
  │ ├── babel-plugin-transform-es2015-object-super@6.24.1 
  │ ├─┬ babel-plugin-transform-es2015-parameters@6.24.1 
  │ │ ├── babel-helper-call-delegate@6.24.1 
  │ │ └── babel-helper-get-function-arity@6.24.1 
  │ ├── babel-plugin-transform-es2015-shorthand-properties@6.24.1 
  │ ├── babel-plugin-transform-es2015-spread@6.22.0 
  │ ├─┬ babel-plugin-transform-es2015-sticky-regex@6.24.1 
  │ │ └── babel-helper-regex@6.26.0 
  │ ├── babel-plugin-transform-es2015-template-literals@6.22.0 
  │ ├── babel-plugin-transform-es2015-typeof-symbol@6.23.0 
  │ ├─┬ babel-plugin-transform-es2015-unicode-regex@6.24.1 
  │ │ └─┬ regexpu-core@2.0.0 
  │ │   ├── regenerate@1.3.3 
  │ │   ├── regjsgen@0.2.0 
  │ │   └─┬ regjsparser@0.1.5 
  │ │     └── jsesc@0.5.0 
  │ ├─┬ babel-plugin-transform-exponentiation-operator@6.24.1 
  │ │ ├─┬ babel-helper-builder-binary-assignment-operator-visitor@6.24.1 
  │ │ │ └── babel-helper-explode-assignable-expression@6.24.1 
  │ │ └── babel-plugin-syntax-exponentiation-operator@6.13.0 
  │ ├─┬ babel-plugin-transform-regenerator@6.26.0 
  │ │ └── regenerator-transform@0.10.1 
  │ └─┬ invariant@2.2.4 
  │   └── loose-envify@1.3.1 
  ├── babel-template@6.26.0 
  ├─┬ babel-traverse@6.26.0 
  │ └── globals@9.18.0 
  ├─┬ babel-types@6.26.0 
  │ └── to-fast-properties@1.0.3 
  ├── babylon@6.18.0 
  ├─┬ babylon-walk@1.0.2 
  │ └── lodash.clone@4.5.0 
  ├─┬ browserslist@2.11.3 
  │ ├── caniuse-lite@1.0.30000823 
  │ └── electron-to-chromium@1.3.41 
  ├─┬ chalk@2.3.2 
  │ ├─┬ ansi-styles@3.2.1 
  │ │ └─┬ color-convert@1.9.1 
  │ │   └── color-name@1.1.3 
  │ ├── escape-string-regexp@1.0.5 
  │ └─┬ supports-color@5.3.0 
  │   └── has-flag@3.0.0 
  ├─┬ chokidar@2.0.3 
  │ ├── anymatch@2.0.0 
  │ ├── async-each@1.0.1 
  │ ├─┬ braces@2.3.1 
  │ │ ├── arr-flatten@1.1.0 
  │ │ ├── define-property@1.0.0 
  │ │ ├─┬ extend-shallow@2.0.1 
  │ │ │ └── is-extendable@0.1.1 
  │ │ ├─┬ fill-range@4.0.0 
  │ │ │ ├── extend-shallow@2.0.1 
  │ │ │ ├─┬ is-number@3.0.0 
  │ │ │ │ └─┬ kind-of@3.2.2 
  │ │ │ │   └── is-buffer@1.1.6 
  │ │ │ ├── repeat-string@1.6.1 
  │ │ │ └── to-regex-range@2.1.1 
  │ │ ├── isobject@3.0.1 
  │ │ ├── repeat-element@1.1.2 
  │ │ ├─┬ snapdragon-node@2.1.1 
  │ │ │ ├── define-property@1.0.0 
  │ │ │ └─┬ snapdragon-util@3.0.1 
  │ │ │   └── kind-of@3.2.2 
  │ │ └── split-string@3.1.0 
  │ ├─┬ glob-parent@3.1.0 
  │ │ ├── is-glob@3.1.0 
  │ │ └── path-dirname@1.0.2 
  │ ├── inherits@2.0.3 
  │ ├─┬ is-binary-path@1.0.1 
  │ │ └── binary-extensions@1.11.0 
  │ ├─┬ is-glob@4.0.0 
  │ │ └── is-extglob@2.1.1 
  │ ├─┬ normalize-path@2.1.1 
  │ │ └── remove-trailing-separator@1.1.0 
  │ ├─┬ readdirp@2.1.0 
  │ │ ├── graceful-fs@4.1.11 
  │ │ └── set-immediate-shim@1.0.1 
  │ └── upath@1.0.4 
  ├── command-exists@1.2.2 
  ├── commander@2.15.1 
  ├─┬ cross-spawn@6.0.5 
  │ ├── nice-try@1.0.4 
  │ ├── path-key@2.0.1 
  │ ├─┬ shebang-command@1.2.0 
  │ │ └── shebang-regex@1.0.0 
  │ └─┬ which@1.3.0 
  │   └── isexe@2.0.0 
  ├─┬ cssnano@3.10.0 
  │ ├─┬ autoprefixer@6.7.7 
  │ │ ├── browserslist@1.7.7 
  │ │ ├── caniuse-db@1.0.30000823 
  │ │ ├── normalize-range@0.1.2 
  │ │ ├── num2fraction@1.2.2 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├── decamelize@1.2.0 
  │ ├── defined@1.0.0 
  │ ├─┬ has@1.0.1 
  │ │ └── function-bind@1.1.1 
  │ ├── object-assign@4.1.1 
  │ ├─┬ postcss@5.2.18 
  │ │ ├─┬ chalk@1.1.3 
  │ │ │ ├── strip-ansi@3.0.1 
  │ │ │ └── supports-color@2.0.0 
  │ │ ├── js-base64@2.4.3 
  │ │ ├── source-map@0.5.7 
  │ │ └─┬ supports-color@3.2.3 
  │ │   └── has-flag@1.0.0 
  │ ├─┬ postcss-calc@5.3.1 
  │ │ ├─┬ postcss@5.2.18 
  │ │ │ ├─┬ chalk@1.1.3 
  │ │ │ │ ├── strip-ansi@3.0.1 
  │ │ │ │ └── supports-color@2.0.0 
  │ │ │ ├── source-map@0.5.7 
  │ │ │ └─┬ supports-color@3.2.3 
  │ │ │   └── has-flag@1.0.0 
  │ │ ├── postcss-message-helpers@2.0.0 
  │ │ └─┬ reduce-css-calc@1.3.0 
  │ │   ├── balanced-match@0.4.2 
  │ │   ├── math-expression-evaluator@1.2.17 
  │ │   └─┬ reduce-function-call@1.0.2 
  │ │     └── balanced-match@0.4.2 
  │ ├─┬ postcss-colormin@2.2.2 
  │ │ ├─┬ colormin@1.1.2 
  │ │ │ ├─┬ color@0.11.4 
  │ │ │ │ ├── clone@1.0.4 
  │ │ │ │ └── color-string@0.3.0 
  │ │ │ └── css-color-names@0.0.4 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-convert-values@2.6.1 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-discard-comments@2.0.4 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-discard-duplicates@2.1.0 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-discard-empty@2.1.0 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-discard-overridden@0.1.1 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-discard-unused@2.2.3 
  │ │ ├─┬ postcss@5.2.18 
  │ │ │ ├─┬ chalk@1.1.3 
  │ │ │ │ ├── strip-ansi@3.0.1 
  │ │ │ │ └── supports-color@2.0.0 
  │ │ │ ├── source-map@0.5.7 
  │ │ │ └─┬ supports-color@3.2.3 
  │ │ │   └── has-flag@1.0.0 
  │ │ └── uniqs@2.0.0 
  │ ├─┬ postcss-filter-plugins@2.0.2 
  │ │ ├─┬ postcss@5.2.18 
  │ │ │ ├─┬ chalk@1.1.3 
  │ │ │ │ ├── strip-ansi@3.0.1 
  │ │ │ │ └── supports-color@2.0.0 
  │ │ │ ├── source-map@0.5.7 
  │ │ │ └─┬ supports-color@3.2.3 
  │ │ │   └── has-flag@1.0.0 
  │ │ └─┬ uniqid@4.1.1 
  │ │   └── macaddress@0.2.8 
  │ ├─┬ postcss-merge-idents@2.1.7 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-merge-longhand@2.0.2 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-merge-rules@2.1.2 
  │ │ ├── browserslist@1.7.7 
  │ │ ├─┬ caniuse-api@1.6.1 
  │ │ │ ├── browserslist@1.7.7 
  │ │ │ ├── lodash.memoize@4.1.2 
  │ │ │ └── lodash.uniq@4.5.0 
  │ │ ├─┬ postcss@5.2.18 
  │ │ │ ├─┬ chalk@1.1.3 
  │ │ │ │ ├── strip-ansi@3.0.1 
  │ │ │ │ └── supports-color@2.0.0 
  │ │ │ ├── source-map@0.5.7 
  │ │ │ └─┬ supports-color@3.2.3 
  │ │ │   └── has-flag@1.0.0 
  │ │ ├─┬ postcss-selector-parser@2.2.3 
  │ │ │ ├── flatten@1.0.2 
  │ │ │ ├── indexes-of@1.0.1 
  │ │ │ └── uniq@1.0.1 
  │ │ └── vendors@1.0.1 
  │ ├─┬ postcss-minify-font-values@1.0.5 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-minify-gradients@1.0.5 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-minify-params@1.2.2 
  │ │ ├── alphanum-sort@1.0.2 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-minify-selectors@2.1.1 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-normalize-charset@1.1.1 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-normalize-url@3.0.8 
  │ │ ├── is-absolute-url@2.1.0 
  │ │ ├─┬ normalize-url@1.9.1 
  │ │ │ ├── prepend-http@1.0.4 
  │ │ │ ├─┬ query-string@4.3.4 
  │ │ │ │ └── strict-uri-encode@1.1.0 
  │ │ │ └─┬ sort-keys@1.1.2 
  │ │ │   └── is-plain-obj@1.1.0 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-ordered-values@2.2.3 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-reduce-idents@2.4.0 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-reduce-initial@1.0.1 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-reduce-transforms@1.0.4 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ ├─┬ postcss-svgo@2.1.6 
  │ │ ├─┬ is-svg@2.1.0 
  │ │ │ └── html-comment-regex@1.1.1 
  │ │ ├─┬ postcss@5.2.18 
  │ │ │ ├─┬ chalk@1.1.3 
  │ │ │ │ ├── strip-ansi@3.0.1 
  │ │ │ │ └── supports-color@2.0.0 
  │ │ │ ├── source-map@0.5.7 
  │ │ │ └─┬ supports-color@3.2.3 
  │ │ │   └── has-flag@1.0.0 
  │ │ └─┬ svgo@0.7.2 
  │ │   ├── coa@1.0.4 
  │ │   ├─┬ csso@2.3.2 
  │ │   │ ├─┬ clap@1.2.3 
  │ │   │ │ └─┬ chalk@1.1.3 
  │ │   │ │   └── strip-ansi@3.0.1 
  │ │   │ └── source-map@0.5.7 
  │ │   ├─┬ js-yaml@3.7.0 
  │ │   │ └── esprima@2.7.3 
  │ │   └── whet.extend@0.9.9 
  │ ├─┬ postcss-unique-selectors@2.0.2 
  │ │ └─┬ postcss@5.2.18 
  │ │   ├─┬ chalk@1.1.3 
  │ │   │ ├── strip-ansi@3.0.1 
  │ │   │ └── supports-color@2.0.0 
  │ │   ├── source-map@0.5.7 
  │ │   └─┬ supports-color@3.2.3 
  │ │     └── has-flag@1.0.0 
  │ └─┬ postcss-zindex@2.2.0 
  │   └─┬ postcss@5.2.18 
  │     ├─┬ chalk@1.1.3 
  │     │ ├── strip-ansi@3.0.1 
  │     │ └── supports-color@2.0.0 
  │     ├── source-map@0.5.7 
  │     └─┬ supports-color@3.2.3 
  │       └── has-flag@1.0.0 
  ├─┬ deasync@0.1.12 
  │ ├── bindings@1.2.1 
  │ └── nan@2.10.0 
  ├── dotenv@5.0.1 
  ├── filesize@3.6.1 
  ├── get-port@3.2.0 
  ├─┬ glob@7.1.2 
  │ ├── fs.realpath@1.0.0 
  │ ├─┬ inflight@1.0.6 
  │ │ └── wrappy@1.0.2 
  │ └── once@1.4.0 
  ├─┬ grapheme-breaker@0.3.2 
  │ ├─┬ brfs@1.5.0 
  │ │ ├─┬ quote-stream@1.0.2 
  │ │ │ ├── buffer-equal@0.0.1 
  │ │ │ └── minimist@1.2.0 
  │ │ ├─┬ static-module@2.2.2 
  │ │ │ ├─┬ concat-stream@1.6.2 
  │ │ │ │ ├── buffer-from@1.0.0 
  │ │ │ │ └── typedarray@0.0.6 
  │ │ │ ├── duplexer2@0.1.4 
  │ │ │ ├─┬ escodegen@1.9.1 
  │ │ │ │ ├── esprima@3.1.3 
  │ │ │ │ ├── estraverse@4.2.0 
  │ │ │ │ └─┬ optionator@0.8.2 
  │ │ │ │   ├── deep-is@0.1.3 
  │ │ │ │   ├── fast-levenshtein@2.0.6 
  │ │ │ │   ├── levn@0.3.0 
  │ │ │ │   ├── prelude-ls@1.1.2 
  │ │ │ │   ├── type-check@0.3.2 
  │ │ │ │   └── wordwrap@1.0.0 
  │ │ │ ├─┬ falafel@2.1.0 
  │ │ │ │ ├── acorn@5.5.3 
  │ │ │ │ ├── foreach@2.0.5 
  │ │ │ │ ├── isarray@0.0.1 
  │ │ │ │ └── object-keys@1.0.11 
  │ │ │ ├─┬ magic-string@0.22.5 
  │ │ │ │ └── vlq@0.2.3 
  │ │ │ ├─┬ merge-source-map@1.0.4 
  │ │ │ │ └── source-map@0.5.7 
  │ │ │ ├── object-inspect@1.4.1 
  │ │ │ ├── shallow-copy@0.0.1 
  │ │ │ └── static-eval@2.0.0 
  │ │ └── through2@2.0.3 
  │ └─┬ unicode-trie@0.3.1 
  │   ├── pako@0.2.9 
  │   └── tiny-inflate@1.0.2 
  ├─┬ htmlnano@0.1.7 
  │ ├─┬ svgo@1.0.5 
  │ │ ├─┬ coa@2.0.1 
  │ │ │ └── q@1.5.1 
  │ │ ├── colors@1.1.2 
  │ │ ├─┬ css-select@1.3.0-rc0 
  │ │ │ ├── boolbase@1.0.0 
  │ │ │ ├── css-what@2.1.0 
  │ │ │ ├── domutils@1.5.1 
  │ │ │ └── nth-check@1.0.1 
  │ │ ├── css-select-base-adapter@0.1.0 
  │ │ ├─┬ css-tree@1.0.0-alpha25 
  │ │ │ ├── mdn-data@1.1.0 
  │ │ │ └── source-map@0.5.7 
  │ │ ├── css-url-regex@1.1.0 
  │ │ ├─┬ csso@3.5.0 
  │ │ │ └─┬ css-tree@1.0.0-alpha.27 
  │ │ │   └── source-map@0.5.7 
  │ │ ├─┬ js-yaml@3.10.0 
  │ │ │ └── esprima@4.0.0 
  │ │ ├─┬ object.values@1.0.4 
  │ │ │ ├── define-properties@1.1.2 
  │ │ │ └─┬ es-abstract@1.11.0 
  │ │ │   ├─┬ es-to-primitive@1.1.1 
  │ │ │   │ ├── is-date-object@1.0.1 
  │ │ │   │ └── is-symbol@1.0.1 
  │ │ │   ├── is-callable@1.1.3 
  │ │ │   └── is-regex@1.0.4 
  │ │ ├── sax@1.2.4 
  │ │ ├── stable@0.1.6 
  │ │ ├── unquote@1.1.1 
  │ │ └─┬ util.promisify@1.0.0 
  │ │   └── object.getownpropertydescriptors@2.0.3 
  │ └─┬ uglify-js@3.3.18 
  │   └── source-map@0.6.1 
  ├── is-url@1.2.4 
  ├─┬ js-yaml@3.11.0 
  │ ├─┬ argparse@1.0.10 
  │ │ └── sprintf-js@1.0.3 
  │ └── esprima@4.0.0 
  ├── json5@0.5.1 
  ├─┬ micromatch@3.1.10 
  │ ├── arr-diff@4.0.0 
  │ ├── array-unique@0.3.2 
  │ ├─┬ define-property@2.0.2 
  │ │ └─┬ is-descriptor@1.0.2 
  │ │   ├── is-accessor-descriptor@1.0.0 
  │ │   └── is-data-descriptor@1.0.0 
  │ ├─┬ extend-shallow@3.0.2 
  │ │ ├── assign-symbols@1.0.0 
  │ │ └─┬ is-extendable@1.0.1 
  │ │   └── is-plain-object@2.0.4 
  │ ├─┬ extglob@2.0.4 
  │ │ ├── define-property@1.0.0 
  │ │ ├─┬ expand-brackets@2.1.4 
  │ │ │ ├─┬ define-property@0.2.5 
  │ │ │ │ └─┬ is-descriptor@0.1.6 
  │ │ │ │   ├─┬ is-accessor-descriptor@0.1.6 
  │ │ │ │   │ └── kind-of@3.2.2 
  │ │ │ │   ├─┬ is-data-descriptor@0.1.4 
  │ │ │ │   │ └── kind-of@3.2.2 
  │ │ │ │   └── kind-of@5.1.0 
  │ │ │ ├── extend-shallow@2.0.1 
  │ │ │ └── posix-character-classes@0.1.1 
  │ │ └── extend-shallow@2.0.1 
  │ ├─┬ fragment-cache@0.2.1 
  │ │ └── map-cache@0.2.2 
  │ ├── kind-of@6.0.2 
  │ ├─┬ nanomatch@1.2.9 
  │ │ ├─┬ is-odd@2.0.0 
  │ │ │ └── is-number@4.0.0 
  │ │ └── is-windows@1.0.2 
  │ ├── object.pick@1.3.0 
  │ ├─┬ regex-not@1.0.2 
  │ │ └─┬ safe-regex@1.1.0 
  │ │   └── ret@0.1.15 
  │ ├─┬ snapdragon@0.8.2 
  │ │ ├─┬ base@0.11.2 
  │ │ │ ├─┬ cache-base@1.0.1 
  │ │ │ │ ├─┬ collection-visit@1.0.0 
  │ │ │ │ │ ├── map-visit@1.0.0 
  │ │ │ │ │ └── object-visit@1.0.1 
  │ │ │ │ ├── get-value@2.0.6 
  │ │ │ │ ├─┬ has-value@1.0.0 
  │ │ │ │ │ └─┬ has-values@1.0.0 
  │ │ │ │ │   └── kind-of@4.0.0 
  │ │ │ │ ├─┬ set-value@2.0.0 
  │ │ │ │ │ └── extend-shallow@2.0.1 
  │ │ │ │ ├─┬ to-object-path@0.3.0 
  │ │ │ │ │ └── kind-of@3.2.2 
  │ │ │ │ ├─┬ union-value@1.0.0 
  │ │ │ │ │ └─┬ set-value@0.4.3 
  │ │ │ │ │   └── extend-shallow@2.0.1 
  │ │ │ │ └─┬ unset-value@1.0.0 
  │ │ │ │   └─┬ has-value@0.3.1 
  │ │ │ │     ├── has-values@0.1.4 
  │ │ │ │     └── isobject@2.1.0 
  │ │ │ ├─┬ class-utils@0.3.6 
  │ │ │ │ ├── arr-union@3.1.0 
  │ │ │ │ ├─┬ define-property@0.2.5 
  │ │ │ │ │ └─┬ is-descriptor@0.1.6 
  │ │ │ │ │   ├─┬ is-accessor-descriptor@0.1.6 
  │ │ │ │ │   │ └── kind-of@3.2.2 
  │ │ │ │ │   ├─┬ is-data-descriptor@0.1.4 
  │ │ │ │ │   │ └── kind-of@3.2.2 
  │ │ │ │ │   └── kind-of@5.1.0 
  │ │ │ │ └─┬ static-extend@0.1.2 
  │ │ │ │   ├─┬ define-property@0.2.5 
  │ │ │ │   │ └─┬ is-descriptor@0.1.6 
  │ │ │ │   │   ├─┬ is-accessor-descriptor@0.1.6 
  │ │ │ │   │   │ └── kind-of@3.2.2 
  │ │ │ │   │   ├─┬ is-data-descriptor@0.1.4 
  │ │ │ │   │   │ └── kind-of@3.2.2 
  │ │ │ │   │   └── kind-of@5.1.0 
  │ │ │ │   └─┬ object-copy@0.1.0 
  │ │ │ │     ├── copy-descriptor@0.1.1 
  │ │ │ │     ├─┬ define-property@0.2.5 
  │ │ │ │     │ └─┬ is-descriptor@0.1.6 
  │ │ │ │     │   ├── is-accessor-descriptor@0.1.6 
  │ │ │ │     │   ├── is-data-descriptor@0.1.4 
  │ │ │ │     │   └── kind-of@5.1.0 
  │ │ │ │     └── kind-of@3.2.2 
  │ │ │ ├── component-emitter@1.2.1 
  │ │ │ ├── define-property@1.0.0 
  │ │ │ ├─┬ mixin-deep@1.3.1 
  │ │ │ │ ├── for-in@1.0.2 
  │ │ │ │ └── is-extendable@1.0.1 
  │ │ │ └── pascalcase@0.1.1 
  │ │ ├─┬ define-property@0.2.5 
  │ │ │ └─┬ is-descriptor@0.1.6 
  │ │ │   ├─┬ is-accessor-descriptor@0.1.6 
  │ │ │   │ └── kind-of@3.2.2 
  │ │ │   ├─┬ is-data-descriptor@0.1.4 
  │ │ │   │ └── kind-of@3.2.2 
  │ │ │   └── kind-of@5.1.0 
  │ │ ├── extend-shallow@2.0.1 
  │ │ ├── source-map@0.5.7 
  │ │ ├─┬ source-map-resolve@0.5.1 
  │ │ │ ├── atob@2.1.0 
  │ │ │ ├── decode-uri-component@0.2.0 
  │ │ │ ├── resolve-url@0.2.1 
  │ │ │ ├── source-map-url@0.4.0 
  │ │ │ └── urix@0.1.0 
  │ │ └── use@3.1.0 
  │ └── to-regex@3.0.2 
  ├─┬ mkdirp@0.5.1 
  │ └── minimist@0.0.8 
  ├── node-forge@0.7.5 
  ├─┬ node-libs-browser@2.1.0 
  │ ├── assert@1.4.1 
  │ ├─┬ browserify-zlib@0.2.0 
  │ │ └── pako@1.0.6 
  │ ├─┬ buffer@4.9.1 
  │ │ ├── base64-js@1.2.3 
  │ │ ├── ieee754@1.1.11 
  │ │ └── isarray@1.0.0 
  │ ├─┬ console-browserify@1.1.0 
  │ │ └── date-now@0.1.4 
  │ ├── constants-browserify@1.0.0 
  │ ├─┬ crypto-browserify@3.12.0 
  │ │ ├─┬ browserify-cipher@1.0.0 
  │ │ │ ├─┬ browserify-aes@1.1.1 
  │ │ │ │ └── buffer-xor@1.0.3 
  │ │ │ ├─┬ browserify-des@1.0.0 
  │ │ │ │ └── des.js@1.0.0 
  │ │ │ └─┬ evp_bytestokey@1.0.3 
  │ │ │   └─┬ md5.js@1.3.4 
  │ │ │     └── hash-base@3.0.4 
  │ │ ├─┬ browserify-sign@4.0.4 
  │ │ │ ├── bn.js@4.11.8 
  │ │ │ ├── browserify-rsa@4.0.1 
  │ │ │ ├─┬ elliptic@6.4.0 
  │ │ │ │ ├── brorand@1.1.0 
  │ │ │ │ ├── hash.js@1.1.3 
  │ │ │ │ ├── hmac-drbg@1.0.1 
  │ │ │ │ ├── minimalistic-assert@1.0.0 
  │ │ │ │ └── minimalistic-crypto-utils@1.0.1 
  │ │ │ └─┬ parse-asn1@5.1.0 
  │ │ │   └── asn1.js@4.10.1 
  │ │ ├── create-ecdh@4.0.0 
  │ │ ├─┬ create-hash@1.1.3 
  │ │ │ ├── cipher-base@1.0.4 
  │ │ │ ├─┬ ripemd160@2.0.1 
  │ │ │ │ └── hash-base@2.0.2 
  │ │ │ └── sha.js@2.4.11 
  │ │ ├── create-hmac@1.1.6 
  │ │ ├─┬ diffie-hellman@5.0.2 
  │ │ │ └── miller-rabin@4.0.1 
  │ │ ├── pbkdf2@3.0.14 
  │ │ ├── public-encrypt@4.0.0 
  │ │ ├── randombytes@2.0.6 
  │ │ └── randomfill@1.0.4 
  │ ├── domain-browser@1.2.0 
  │ ├── events@1.1.1 
  │ ├── https-browserify@1.0.0 
  │ ├── os-browserify@0.3.0 
  │ ├── path-browserify@0.0.0 
  │ ├── process@0.11.10 
  │ ├── punycode@1.4.1 
  │ ├── querystring-es3@0.2.1 
  │ ├─┬ readable-stream@2.3.5 
  │ │ ├── core-util-is@1.0.2 
  │ │ ├── process-nextick-args@2.0.0 
  │ │ └── util-deprecate@1.0.2 
  │ ├── stream-browserify@2.0.1 
  │ ├─┬ stream-http@2.8.1 
  │ │ ├── builtin-status-codes@3.0.0 
  │ │ ├── to-arraybuffer@1.0.1 
  │ │ └── xtend@4.0.1 
  │ ├── string_decoder@1.0.3 
  │ ├─┬ timers-browserify@2.0.6 
  │ │ └── setimmediate@1.0.5 
  │ ├── tty-browserify@0.0.0 
  │ ├─┬ url@0.11.0 
  │ │ ├── punycode@1.3.2 
  │ │ └── querystring@0.2.0 
  │ ├─┬ util@0.10.3 
  │ │ └── inherits@2.0.1 
  │ └─┬ vm-browserify@0.0.4 
  │   └── indexof@0.0.1 
  ├─┬ opn@5.3.0 
  │ └── is-wsl@1.1.0 
  ├── physical-cpu-count@2.0.0 
  ├─┬ postcss@6.0.21 
  │ └── supports-color@5.3.0 
  ├── postcss-value-parser@3.3.0 
  ├─┬ posthtml@0.11.3 
  │ └─┬ posthtml-parser@0.3.3 
  │   └── isobject@2.1.0 
  ├─┬ posthtml-parser@0.4.1 
  │ └─┬ htmlparser2@3.9.2 
  │   ├── domelementtype@1.3.0 
  │   ├── domhandler@2.4.1 
  │   ├─┬ domutils@1.7.0 
  │   │ └─┬ dom-serializer@0.1.0 
  │   │   └── domelementtype@1.1.3 
  │   └── entities@1.1.1 
  ├── posthtml-render@1.1.2 
  ├─┬ resolve@1.6.0 
  │ └── path-parse@1.0.5 
  ├── semver@5.5.0 
  ├─┬ serialize-to-js@1.2.0 
  │ ├─┬ js-beautify@1.7.5 
  │ │ ├─┬ config-chain@1.1.11 
  │ │ │ ├── ini@1.3.5 
  │ │ │ └── proto-list@1.2.4 
  │ │ ├─┬ editorconfig@0.13.3 
  │ │ │ ├── bluebird@3.5.1 
  │ │ │ ├─┬ lru-cache@3.2.0 
  │ │ │ │ └── pseudomap@1.0.2 
  │ │ │ └── sigmund@1.0.1 
  │ │ └─┬ nopt@3.0.6 
  │ │   └── abbrev@1.1.1 
  │ └─┬ safer-eval@1.2.3 
  │   └── clones@1.1.0 
  ├─┬ serve-static@1.13.2 
  │ ├── encodeurl@1.0.2 
  │ ├── escape-html@1.0.3 
  │ ├── parseurl@1.3.2 
  │ └─┬ send@0.16.2 
  │   ├── depd@1.1.2 
  │   ├── destroy@1.0.4 
  │   ├── etag@1.8.1 
  │   ├── fresh@0.5.2 
  │   ├─┬ http-errors@1.6.3 
  │   │ └── setprototypeof@1.1.0 
  │   ├── mime@1.4.1 
  │   ├─┬ on-finished@2.3.0 
  │   │ └── ee-first@1.1.1 
  │   ├── range-parser@1.2.0 
  │   └── statuses@1.4.0 
  ├── source-map@0.6.1 
  ├─┬ strip-ansi@4.0.0 
  │ └── ansi-regex@3.0.0 
  ├── toml@2.3.3 
  ├── tomlify-j0.4@3.0.0 
  ├─┬ uglify-es@3.3.9 
  │ └── commander@2.13.0 
  ├── v8-compile-cache@1.1.2 
  ├─┬ worker-farm@1.6.0 
  │ └─┬ errno@0.1.7 
  │   └── prr@1.0.1 
  └─┬ ws@4.1.0 
    ├── async-limiter@1.0.0 
    └── safe-buffer@5.1.1 

npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.1.3

```
but npm start fails

```
⋊> ~/g/3/walletconnect.js on master ◦ npm start                         13:14:10

> walletconnect.js@1.0.0 start /home/ligi/git/3rd/walletconnect.js
> parcel src/html/index.html -p 4444 --open

(node:11701) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: Cannot find module './SourceMapPackager'

```

--- a/chains/ethereum/package.json
+++ b/chains/ethereum/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@web3modal/ethereum",
-  "version": "2.1.2",
+  "version": "2.1.3",
   "main": "dist/index.js",
   "type": "module",
   "types": "dist/_types/index.d.ts",

--- a/examples/create-react-app/package.json
+++ b/examples/create-react-app/package.json
@@ -1,10 +1,10 @@
 {
   "name": "cra-example",
-  "version": "2.1.2",
+  "version": "2.1.3",
   "private": true,
   "dependencies": {
-    "@web3modal/ethereum": "2.1.2",
-    "@web3modal/react": "2.1.2",
+    "@web3modal/ethereum": "2.1.3",
+    "@web3modal/react": "2.1.3",
     "ethers": "5.7.2",
     "react": "18.2.0",
     "react-dom": "18.2.0",

--- a/examples/html-standalone/package.json
+++ b/examples/html-standalone/package.json
@@ -1,7 +1,7 @@
 {
   "name": "html-standalone-example",
   "private": true,
-  "version": "2.1.2",
+  "version": "2.1.3",
   "type": "module",
   "scripts": {
     "dev": "vite",
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "@walletconnect/sign-client": "2.4.4",
-    "@web3modal/standalone": "2.1.2"
+    "@web3modal/standalone": "2.1.3"
   },
   "devDependencies": {
     "vite": "4.1.3"

--- a/examples/html/package.json
+++ b/examples/html/package.json
@@ -1,7 +1,7 @@
 {
   "name": "html-example",
   "private": true,
-  "version": "2.1.2",
+  "version": "2.1.3",
   "type": "module",
   "scripts": {
     "dev": "vite",
@@ -10,8 +10,8 @@
   "dependencies": {
     "@gnosis.pm/safe-apps-wagmi": "2.1.0",
     "@wagmi/core": "0.9.6",
-    "@web3modal/ethereum": "2.1.2",
-    "@web3modal/html": "2.1.2",
+    "@web3modal/ethereum": "2.1.3",
+    "@web3modal/html": "2.1.3",
     "ethers": "5.7.2"
   },
   "devDependencies": {

--- a/examples/laboratory/package.json
+++ b/examples/laboratory/package.json
@@ -1,6 +1,6 @@
 {
   "name": "laboratory",
-  "version": "2.1.2",
+  "version": "2.1.3",
   "private": true,
   "scripts": {
     "dev": "next dev",
@@ -10,8 +10,8 @@
     "@nextui-org/react": "1.0.0-beta.12",
     "@sentry/react": "7.38.0",
     "@sentry/tracing": "7.38.0",
-    "@web3modal/ethereum": "2.1.2",
-    "@web3modal/react": "2.1.2",
+    "@web3modal/ethereum": "2.1.3",
+    "@web3modal/react": "2.1.3",
     "ethers": "5.7.2",
     "next": "13.1.6",
     "react": "18.2.0",

--- a/examples/nextjs-standalone/package.json
+++ b/examples/nextjs-standalone/package.json
@@ -1,13 +1,13 @@
 {
   "name": "nextjs-standalone-example",
-  "version": "2.1.2",
+  "version": "2.1.3",
   "private": true,
   "scripts": {
     "dev": "next dev"
   },
   "dependencies": {
     "@walletconnect/sign-client": "2.4.4",
-    "@web3modal/standalone": "2.1.2",
+    "@web3modal/standalone": "2.1.3",
     "next": "13.1.6",
     "react": "18.2.0",
     "react-dom": "18.2.0"

--- a/examples/nextjs/package.json
+++ b/examples/nextjs/package.json
@@ -1,13 +1,13 @@
 {
   "name": "nextjs-example",
-  "version": "2.1.2",
+  "version": "2.1.3",
   "private": true,
   "scripts": {
     "dev": "next dev"
   },
   "dependencies": {
-    "@web3modal/ethereum": "2.1.2",
-    "@web3modal/react": "2.1.2",
+    "@web3modal/ethereum": "2.1.3",
+    "@web3modal/react": "2.1.3",
     "ethers": "5.7.2",
     "next": "13.1.6",
     "react": "18.2.0",

--- a/lerna.json
+++ b/lerna.json
@@ -9,5 +9,5 @@
     "packages/standalone"
   ],
   "useWorkspaces": true,
-  "version": "2.1.2"
+  "version": "2.1.3"
 }

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@web3modal/core",
-  "version": "2.1.2",
+  "version": "2.1.3",
   "main": "dist/index.js",
   "type": "module",
   "types": "dist/_types/index.d.ts",
@@ -17,10 +17,10 @@
   },
   "dependencies": {
     "buffer": "6.0.3",
-    "valtio": "1.10.2"
+    "valtio": "1.9.0"
   },
   "devDependencies": {
-    "@web3modal/ethereum": "2.1.2"
+    "@web3modal/ethereum": "2.1.3"
   },
   "keywords": [
     "web3",

--- a/packages/html/package.json
+++ b/packages/html/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@web3modal/html",
-  "version": "2.1.2",
+  "version": "2.1.3",
   "main": "dist/index.js",
   "type": "module",
   "types": "dist/_types/index.d.ts",
@@ -16,11 +16,11 @@
     "typecheck": "tsc --noEmit"
   },
   "devDependencies": {
-    "@web3modal/ethereum": "2.1.2"
+    "@web3modal/ethereum": "2.1.3"
   },
   "dependencies": {
-    "@web3modal/core": "2.1.2",
-    "@web3modal/ui": "2.1.2"
+    "@web3modal/core": "2.1.3",
+    "@web3modal/ui": "2.1.3"
   },
   "keywords": [
     "web3",

--- a/packages/react/package.json
+++ b/packages/react/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@web3modal/react",
-  "version": "2.1.2",
+  "version": "2.1.3",
   "main": "dist/index.js",
   "type": "module",
   "types": "dist/_types/index.d.ts",
@@ -16,13 +16,13 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
-    "@web3modal/core": "2.1.2",
-    "@web3modal/ui": "2.1.2"
+    "@web3modal/core": "2.1.3",
+    "@web3modal/ui": "2.1.3"
   },
   "devDependencies": {
     "@types/react": "18.0.28",
     "@types/react-dom": "18.0.11",
-    "@web3modal/ethereum": "2.1.2",
+    "@web3modal/ethereum": "2.1.3",
     "eslint-plugin-react": "7.32.2",
     "eslint-plugin-react-hooks": "4.6.0",
     "react": "18.2.0",

--- a/packages/standalone/package.json
+++ b/packages/standalone/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@web3modal/standalone",
-  "version": "2.1.2",
+  "version": "2.1.3",
   "main": "dist/index.js",
   "type": "module",
   "types": "dist/_types/index.d.ts",
@@ -16,8 +16,8 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
-    "@web3modal/core": "2.1.2",
-    "@web3modal/ui": "2.1.2"
+    "@web3modal/core": "2.1.3",
+    "@web3modal/ui": "2.1.3"
   },
   "keywords": [
     "web3",

--- a/packages/ui/package.json
+++ b/packages/ui/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@web3modal/ui",
-  "version": "2.1.2",
+  "version": "2.1.3",
   "main": "dist/index.js",
   "type": "module",
   "types": "dist/_types/index.d.ts",
@@ -16,7 +16,7 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
-    "@web3modal/core": "2.1.2",
+    "@web3modal/core": "2.1.3",
     "lit": "2.6.1",
     "motion": "10.15.5",
     "qrcode": "1.5.1"

